packages/api/src/command/usage/report-usage.ts

```diff
@@ -1,24 +1,20 @@
 import Axios from "axios";
+import { Product } from "../../domain/product";
 import { Config } from "../../shared/config";
 import { capture } from "../../shared/notifications";
 
-const axios = Axios.create({ timeout: 10_000 });
-
-export enum ApiTypes {
-  devices = "devices",
-  medical = "medical",
-}
+const axios = Axios.create({ timeout: 20_000 });
```

Comment:
Increased the timeout b/c I've seen some situations where it times out when reporting to internal. Since it's I/O and Node is doing something in the meanwhile, I reckoned it would be fine to increase this.